src/core/iomgr/iomgr.h

```diff
@@ -73,4 +73,6 @@ void grpc_iomgr_shutdown(void);
  * Can be called from within a callback or from anywhere else */
 void grpc_iomgr_add_callback(grpc_iomgr_closure *closure);
 
+void grpc_iomgr_add_delayed_callback(grpc_iomgr_closure *iocb, int success);
```

Comment:
Is there a distinction between the use case for these two functions? If so, the second also needs a comment.